modules/exploits/windows/http/meddream_pacs_rce.rb

```diff
@@ -0,0 +1,96 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'MedDream PacsServer Unauthenticated RCE',
+      'Description' => %q{
+        This module exploits a vulnerability found in MedDream PacsServer 6.8.3.751,
+        which allows unauthenticated attackers to perform Remote Code Execution
+        via an arbitrary PHP file upload.
+      },
+      'License' => MSF_LICENSE,
+      'Author' =>
+        [
+        'bzyo'
+        ],
+      'Platform' => 'php',
+      'References' =>
+          [
+            ['EDB', '48868'],
```

Comment:
i didn't see the company put out anything, did they have anything in release notes?